main.tf

```diff
@@ -5,7 +5,8 @@ locals {
   watsonx_orchestrate_guid          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].guid : ibm_resource_instance.watsonx_orchestrate_instance[0].guid
   watsonx_orchestrate_name          = var.existing_watsonx_orchestrate_instance_crn != null ? data.ibm_resource_instance.existing_orchestrate_instance[0].resource_name : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_name
   watsonx_orchestrate_plan_id       = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].resource_plan_id
-  watsonx_orchestrate_dashboard_url = var.existing_watsonx_orchestrate_instance_crn != null ? null : ibm_resource_instance.watsonx_orchestrate_instance[0].dashboard_url
+  raw_crn                           = "crn:v1:bluemix:public:watsonx-orchestrate:${var.region}:a/${local.account_id}:${local.watsonx_orchestrate_guid}::"
```

Comment:
Provide a comment mentioning this is a temporary workaround and provide issue number so that it is easy to understand why instead of directly using `dashboard_url` output we are customizing URL